testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -498,6 +498,15 @@ def test_slicing_step_with_start_stop():
         assert_equal(universe.trajectory.timeseries(universe.atoms).shape,
                      (3341, 10, 3),
                      err_msg="Unexpected shape of trajectory timeseries")
+    @staticmethod
+    def test_slicing_step_dt():
+        universe = MDAnalysis.Universe(PDB_small, DCD)
```

Comment:
Do you suggest I switch? I thought about it, but ended up following the other test_slicing_* methods above (and below) mine.